doc/architecture/glossary.md

```diff
@@ -29,19 +29,35 @@
   From the seeds of a fruit you get again similar fruits.
 - raising (German: Anzucht, vorziehen)
 - (German: Vorkulturen)
+- Taxonomy:
+  A scientific classification of plants into hierarchical groups.
```

Comment:
This is a bit sloppy, better to link to Wikipedia or similar.

Reply:
See https://github.com/ElektraInitiative/PermaplanT/pull/250/commits/fd85a7a996ed3e3c8930843713bb37593c5220cb